Makefile

```diff
@@ -2,7 +2,8 @@
 
 ## Important paths ####################################
 # This is where all tests are run
-SCRATCH_SPACE 		= /home/rbasuro/riscv-new-scratch
+TOP = /afs/eos.ncsu.edu/lockers/research/ece/ericro/users/mmkarand
```

Comment:
Do you need the TOP variable? It doesn't seem to be used anywhere. Moreover, TOP isn't a very intuitive name.

Reply:
It is not a necessary variable. It is used in the next line just to define scratch space. Although this would have to changed everytime for different user working on this project. TOP just meant the folder where the anycore project resides. No other significance. Should I remove it?

Reply:
Yes, please remove it. I can merge after that.

Reply:
In my latest commit I have removed the TOP variable and replaced it with single SCRATCH_SPACE variable.